packages/web-app-admin-settings/src/components/Users/GroupsModal.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <oc-modal
+    :title="title"
+    :message="message"
+    :button-cancel-text="$gettext('Cancel')"
+    :button-confirm-text="$gettext('Confirm')"
+    :button-confirm-disabled="!selectedOptions.length"
+    @cancel="$emit('cancel')"
+    @confirm="$emit('confirm', { users, groups: selectedOptions })"
+  >
+    <template #content>
+      <oc-select
+        ref="groupsModal"
+        v-model="selectedOptions"
+        :label="$gettext('Groups')"
+        multiple
+        option-label="displayName"
+        :options="groups"
+      />
+    </template>
+  </oc-modal>
+</template>
+
+<script lang="ts">
+import { defineComponent, PropType, ref } from 'vue'
+import { Group, User } from 'web-client/src/generated'
+
+export default defineComponent({
+  name: 'GroupsModal',
+  props: {
+    title: {
+      type: String,
+      required: true
+    },
+    message: {
+      type: String,
+      required: true
+    },
+    groups: {
```

Comment:
Please sort the groups in the same manner as in the right sidebar user edit form.